tests/unit/tasks/server/livereload-server-test.js

```diff
@@ -75,6 +75,16 @@ describe('livereload-server', function() {
           preexistingServer.close(done);
         });
     });
+
+    it('starts with custom host', function() {
+      return subject.start({
+        liveReloadHost: '127.0.0.1',
```

Comment:
ah, good point. 127.0.0.1 should be fine.

If we could assume IPV6 support across all our test runners, `::1` may be an option.